product_set/wizard/product_set_add.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2015 Anybox S.A.S
+# Copyright 2016-2020 Camptocamp SA
+# @author Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class ProductSetAdd(models.TransientModel):
```

Comment:
Shouldn't this be an AbstractModel in the style of a mixin?

Reply:
done, thanks!